api/src/main/scala/za/co/absa/loginsvc/rest/AuthManagerConfig.scala

```diff
@@ -36,23 +36,22 @@ import scala.collection.immutable.SortedMap
  * @param authConfigsProvider
  */
 @Configuration
-class AuthManagerConfig @Autowired()(authConfigsProvider: AuthConfigProvider){
+class AuthManagerConfig @Autowired()(authConfigsProvider: AuthConfigProvider, objectProcessor: ObjectPostProcessor[Object]){
 
   private val usersConfig: Option[UsersConfig] = authConfigsProvider.getUsersConfig
   private val adLDAPConfig: Option[ActiveDirectoryLDAPConfig] = authConfigsProvider.getLdapConfig
 
   private val logger = LoggerFactory.getLogger(classOf[AuthManagerConfig])
 
   @Bean
-  def authManager(http: HttpSecurity): AuthenticationManager = {
-
-    val authenticationManagerBuilder = http.getSharedObject(classOf[AuthenticationManagerBuilder]).parentAuthenticationManager(null)
+  def authManager(): AuthenticationManager = {
     val configs: Array[ConfigOrdering] = Array(usersConfig, adLDAPConfig).flatten
     val orderedProviders = createAuthProviders(configs)
 
     if(orderedProviders.isEmpty)
       throw ConfigValidationException("No authentication method enabled in config")
 
+    val authenticationManagerBuilder = new AuthenticationManagerBuilder(objectProcessor)
```

Comment:
This change was necessary, otherwise the there was an error that the underlying AuthManager is null.